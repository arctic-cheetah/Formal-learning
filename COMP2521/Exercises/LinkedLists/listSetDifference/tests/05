9 1 8 2 5
1 5 2 9 8
