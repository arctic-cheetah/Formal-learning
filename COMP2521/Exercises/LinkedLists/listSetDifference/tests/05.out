Enter list 1: Enter list 2: 
Set 1: {9, 1, 8, 2, 5}
Set 2: {1, 5, 2, 9, 8}
Difference: {}
